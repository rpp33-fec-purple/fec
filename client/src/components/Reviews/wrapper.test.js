
import React from 'react';
import TestRenderer from 'react-test-renderer';
import Review from './wrapper';

describe('render the component RelatedItems', () => {
const testRenderer = TestRenderer.create(<Review />);
const testInst = testRenderer.root;
it('should contain a h2 element ', () => {
  expect(testRenderer.toJSON().type).toBe('h2');
})

it('should return the right inner text of the existing element ', () => {
  expect(testInst.findByProps({className:'Review'}).children[0]).toEqual('Review Component');
})

// it('the component should match the saved snapshot', () => {
//   expect(testRenderer.toJSON()).toMatchSnapshot();
// })

})

// it('render the component Q&A', () => {
//   const testRenderer = TestRenderer.create(<QA />);
//   expect(testRenderer.toJSON().children[0]).toEqual('Questions and Answers Component')
//   expect(testRenderer.toJSON()).toMatchSnapshot();
// })