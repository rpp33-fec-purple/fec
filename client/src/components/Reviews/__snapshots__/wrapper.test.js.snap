// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the component Reviews the component should match the saved snapshot 1`] = `
<div>
  <h4
    className="Review"
  >
    RATINGS & REVIEWS
  </h4>
  <div>
    <div />
    <button>
      MORE REVIEWS
    </button>
    <button>
      ADD A REVIEW +
    </button>
  </div>
  <div>
    <div>
      No Reviews Yet
    </div>
  </div>
  <div />
</div>
`;
